gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/amqp/ServiceFailureIT.java

```diff
@@ -110,7 +116,7 @@ public void When_SendingInitialRequest_WithMhsOutboundServerError_Expect_Migrati
 
         sendRequestToPssQueue(conversationId, patientNhsNumber);
 
-        await().until(() -> hasMigrationStatus(EHR_EXTRACT_REQUEST_ERROR, conversationId));
+        await().atMost(Duration.ofMinutes(TWO_MINUTES_LONG)).until(() -> hasMigrationStatus(EHR_EXTRACT_REQUEST_ERROR, conversationId));
```

Comment:
Does this need to be longer because of the retries now? How long does it wait between retries?